### PR TITLE
Try to run NL integration tests against staging

### PR DESCRIPTION
**Pull request by pradh**

### CODE REVIEW

**Review by shifucun — APPROVED** [review]
thanks for the update!

File: server/app_env/integration_test.py [hunk 1]
```diff
@@ -20,4 +20,10 @@ class Config(_base.Config):
   API_ROOT = 'https://autopush.api.datacommons.org'
```
Comment (shifucun) [inline server/app_env/integration_test.py:20]: should make this AutopushConfig then?

**Review by shifucun — APPROVED** [review]

File: server/app_env/integration_test.py (hunk 1 above)
Comment (pradh) [reply server/app_env/integration_test.py:20]: Done.

Comment (shifucun) [reply server/app_env/integration_test.py:20]: Also update below to be StagingConfig(base._Config): with 5 fields defined?

Comment (pradh) [reply server/app_env/integration_test.py:20]: Thanks!  Indeed, I badly broke tests.

**Pull request merged**